.github/workflows/bc-check.yml

```diff
@@ -0,0 +1,18 @@
+
+on:
+  pull_request:
+  push:
```

Comment:
Should probably match `main` branch wdyt?

Reply:
Yes, Correct. 

I see that you took a different direction than the one I suggested in #26. 

I've updated the PR. Thank you